apps/settings/js/panels/feedback_send/feedback_send.js

```diff
@@ -136,9 +141,6 @@ define(function(require) {
     },
 
     _responseHandler: function() {
-      if (this._xhr.readyState !== 4) {
-        return;
-      }
       switch (this._xhr.status) {
```

Comment:
Ok, so for `case 400` and `default` case, I think the better way is to show a general error message like **oops, something went wrong and please try again**, what do you think ? Because there are too many cases here and we can't even list all them down, the better way is to show this message.

And for `429` case, let's follow your comment and treat that request as success and don't mislead users here